fiftyone/core/models.py

```diff
@@ -486,8 +486,9 @@ def _apply_image_model_data_loader(
             )
         )
 
-        def save_batch(sample_batch, labels_batch):
+        def save_batch(sample_batch, model, output, image_sizes):
             with _handle_batch_error(skip_failures, sample_batch):
+                labels_batch = model.postprocess(output, image_sizes)
```

Comment:
Unless I'm missing something, this will cause `apply_model()` to start failing for all existing `TorchImageModel` instances besides `FiftyOneTransformer` which you've specifically implemented `postprocess()` for. We can't do that.

For context, the model zoo contains lots of instances of `TorchvisionImageModel(TorchImageModel)`. [EG here](https://github.com/voxel51/fiftyone/blob/2853b72cb393c1b08d1f14bcfd06da5ee7d3ae50/fiftyone/zoo/models/manifest-torch.json#L19)